noir-projects/noir-contracts/contracts/token_with_refunds_contract/src/main.nr

```diff
@@ -535,6 +527,7 @@ contract TokenWithRefunds {
         let tx_fee = U128::from_integer(context.transaction_fee());
 
         // 1. We check that user funded the fee payer contract with at least the transaction fee.
+        // TODO(#7796): we should try to prevent reverts here
```

Comment:
Created [an issue](https://github.com/AztecProtocol/aztec-packages/issues/7796) and linked it here to not forget.